Add files of flask tutorial

zhaoqix2 opened the pull request.

zhaoqix2 committed:
Add files of flask tutorial

meghanreynolds commented on Research/zhaoqix2/flaskr/__init__.py:
```diff
@@ -0,0 +1,18 @@
+def greeting():
+    return 'Have a nice day!'
+
+@app.route('/hello/Zhaoqi')
+def helloPerson():
+    return 'Hello Zhaoqi!'
```
https://flask.palletsprojects.com/en/1.1.x/quickstart/#variable-rules may help with completing the third part of the tutorial

meghanreynolds reviewed (commented):
Great work so far!

mschulz285 committed:
Created hello.py

Followed the Quickstart page (https://flask.palletsprojects.com/en/1.1.x/quickstart/) and created and tested my Flask page, following the instructions in FlaskTutorial.md.

meghanreynolds committed:
CHORE: Removed Extra File
